src/encoded/static/components/genome_browser.js

```diff
@@ -879,7 +895,8 @@ class GenomeBrowser extends React.Component {
             // Here we do some approximate math to try to figure out how many lines the labels extend to assuming that ~30 characters fit on one line
             // Labels on the experiment pages are short enough to fit on one line (they contain less information) so we can bypass these calculations for those pages
             if (label === 'cart') {
-                labelLength += file.target && file.target.label ? file.target.label.length + 2 : 0;
+                const fileTarget = getFileTarget(file);
+                labelLength += fileTarget && fileTarget.label ? fileTarget.label.length + 2 : 0;
```

Comment:
+= on strings is code smell and it is better to use a join given that strings are immutable so += is doing extra work to concatenate strings. But I guess since the work is small it is no big deal.